reader-osm/src/test/resources/com/graphhopper/reader/osm/test-conditional-turn-restrictions.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<osm version="0.6" generator="pbf2osm">
+    <!-- 
+       1~~~2
+       |\  |
+       | 6 3~~8
+       |/  |
+       5~~~4~~7
+       
+       
+      (2,3)->(3,4): straight_only
+      (4,3)->(3,8): no_right_turn
+      (4,5)->(5,6): right_turn_only
+      (6,1)->(1,6): no_u_turn
+    -->
+
+    <node id="1" lat="50" lon="10"> </node>
+    <node id="2" lat="52" lon="10"> </node>
+    <node id="3" lat="52" lon="11"> </node>
+    <node id="4" lat="52" lon="12"> </node>
+    <node id="5" lat="50" lon="12"> </node>
+    <node id="6" lat="51" lon="11"> </node>
+    <node id="7" lat="54" lon="12"> </node>
+    <node id="8" lat="54" lon="11"> </node>
+                      
+    <!-- 1-2 -->
+    <way id="12">
+        <nd ref="1"/>
+        <nd ref="2"/>
+        <tag k="highway" v="motorway" />
+    </way>    
+    
+    <!-- 2-3 -->
+    <way id="23">
+        <nd ref="2"/>        
+        <nd ref="3"/>
+        <tag k="highway" v="motorway" />
+    </way>
+    
+    <!-- 3-4 -->
+    <way id="34">
+        <nd ref="3"/>     
+        <nd ref="4"/>
+        <tag k="highway" v="motorway" />
+    </way>     
+    
+    <!-- 4-5 -->
+    <way id="45">
+        <nd ref="5"/>     
+        <nd ref="4"/>
+        <tag k="highway" v="motorway" />
+    </way>  
+    
+    <!-- 5-1 -->
+    <way id="51">
+        <nd ref="1"/>     
+        <nd ref="5"/>
+        <tag k="highway" v="motorway" />
+    </way>
+    
+    <!-- 5-6 -->
+    <way id="56">
+        <nd ref="6"/>     
+        <nd ref="5"/>
+        <tag k="highway" v="motorway" />
+    </way>
+    
+    <!-- 6-1 -->
+    <way id="61">
+        <nd ref="1"/>     
+        <nd ref="6"/>
+        <tag k="highway" v="motorway" />
+    </way>
+    
+    <!-- 3-8 -->
+    <way id="38">
+        <nd ref="3"/>     
+        <nd ref="8"/>
+        <tag k="highway" v="motorway" />
+    </way>
+    
+    <!-- 4-7 -->
+    <way id="47">
+        <nd ref="4"/>     
+        <nd ref="7"/>
+        <tag k="highway" v="motorway" />
+    </way>
+    
+    <way id="510">
+        <nd ref="5"/>     
+        <nd ref="10"/>
+        <tag k="highway" v="cycleway" />
+    </way>
+        
+    <relation id="1001">
+        <member type="way" ref="23" role="from"/>
+        <member type="way" ref="34" role="to"/>
+        <member type="node" ref="3" role="via"/>
+        <tag k="restriction" v="only_straight_on"/>
```

Comment:
We should add a test that covers the case that previously failed: A relation that has multiple conditional restrictions. Maybe also covering the case where there are multiple conditional restrictions and an except list. We could do this here, but usually I think its better to have isolated tests for the different cases (they can be as small as a single junction ?)